package.json

```
@@ -13,39 +13,42 @@
   },
   "private": true,
```

If you use ES syntax in `generate.js`, you should set package as module to run `yarn lib:generate`
```suggestion
  "private": true,
  "type":"module",
```